process/transactionProcessor.go

```diff
@@ -538,7 +538,12 @@ func isZeroValue(value string) bool {
 	return value == "0"
 }
 
-func checkIfFailed(logs []*transaction.ApiLogs) (bool, string) {
+func checkIfFailed(tx *transaction.ApiTransactionResult, logs []*transaction.ApiLogs) (bool, string) {
+	// if relayed v3, skip this check. inner transactions can fail
```

Comment:
_"are allowed to fail without considering the parent transaction as completely failed"_ - or something similar.